src/clientes/clientes.class.ts

```diff
@@ -0,0 +1,14 @@
+import { Conta } from "src/contas/conta.class";
+
+export class Cliente {
+  id: string;
+  contas: any;
+  constructor(
+    id: string,
+    nomeCompleto: string,
+    numeroIdentificacao: string,
+    endereco: string,
+    telefone: string,
+    rendaSalarial: number,
+    contas: Conta[],) { }
+}
```

Comment:
Aqui no constructor você está utilizando alguns atributos que não foram declarados fora dele. Uma sugestão seria: 

-  ou declarar os atributos de dentro do constructor como public e retirar esse id e contas do lado de fora
- ou declarar todos eles fora do constructor e voltar a escrevê-los dentro dele e dentro das chaves colocar o this.(atributo) = atributo.